### PR TITLE
Update declaration questions for DOS 5

### PR DESCRIPTION
**Pull request by bjgill**
Trello: https://trello.com/c/yWkgqyPL/325-2-update-dos5-application-content-in-the-frameworks-repo

We need to make a number of changes following the accessibility review and CCS requests. See the spreadsheet linked from the Trello card for details.

This PR now covers all the declaration changes requested, except the reports of error oddities. I've not been able to reproduce them. Let's get this into Preview and see if they show up there.

Don't try and review this all together. I'd recommend going through commit by commit instead - they're all independent of each other.

### CODE REVIEW

**Comment by lfdebrux** [comment]
### My review checklist

I've:

- [ ] looked at the code
- [ ] tested locally
- [ ] run functional tests locally

**Comment by lfdebrux** [comment]
There are a couple of places where we use 'opens new tab' instead of 'opens in new tab', should we change those to be consistent with the new content?

```
$ rg 'opens new tab' frameworks/digital-outcomes-and-specialists-5
frameworks/digital-outcomes-and-specialists-5/questions/declaration/modernSlaveryStatement.yml
10:  <a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/accessible-pdfs" target="_blank" rel="noopener noreferrer">Read the guidance on accessible documents (opens new tab)</a>.
12:  <a href="https://www.gov.uk/government/publications/transparency-in-supply-chains-a-practical-guide" target="_blank" rel="noopener noreferrer">Read the Home Office guidance about how to write a statement and what information to include (opens new tab)</a>.

frameworks/digital-outcomes-and-specialists-5/questions/declaration/modernSlaveryStatementOptional.yml
13:  <a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/accessible-pdfs" target="_blank" rel="noopener noreferrer">Read the guidance on accessible documents (opens new tab)</a>.
15:  <a href="https://www.gov.uk/government/publications/transparency-in-supply-chains-a-practical-guide" target="_blank" rel="noopener noreferrer">Read the Home Office guidance about how to write a statement and what information to include (opens new tab)</a>.
```

File: frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml [hunk 1]
```diff
@@ -173,12 +173,12 @@
   name: Contact details
   slug: contact-details
   editable: True
-  prefill: True
+  prefill: False
   description: |
      The Crown Commercial Service will use the primary contact information to contact your organisation if it has any questions about your application.
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml:179]: Do we want to remove 'primary' from this content as well?

```suggestion
     The Crown Commercial Service will use the contact information to contact your organisation if it has any questions about your application.
```

**Comment by bjgill** [comment]
> There are a couple of places where we use 'opens new tab' instead of 'opens in new tab', should we change those to be consistent with the new content?

@NoraGDS I can see both 'opens new tab' and 'opens in new tab' in the spreadsheet. Which is the preferred form?

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml [hunk 2]
```diff
@@ -1,8 +1,14 @@
 name: Understanding the application process
-question: >
-  Do you understand the application process for Digital Outcomes and Specialists&nbsp;5 on the Digital Marketplace?
-hint: >
+question: |
   You can read about how to apply in the <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide#how-to-apply" target="_blank" rel="noopener noreferrer">Digital Outcomes and Specialists suppliers’ guide (opens in new tab)</a>.
+
+  By starting an application for Digital Outcomes and Specialists&nbsp;5 do you understand:
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml:5]: Is this the correct content? It doesn't quite scan for me 🤔

File: frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml [hunk 3]
```diff
@@ -127,21 +135,6 @@
...
-  description: |
-    You must be able to truthfully answer ‘yes’ to every question on this page for your application to be considered eligible.
-
-    If you can’t answer ‘yes’ to every question on this page, it’s very unlikely that your application will be accepted.
-
-  questions:
-    - readUnderstoodGuidance
-    - understandTool
-    - understandHowToAskQuestions
-
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml:151]: 🔥

File: frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml [hunk 4]
```diff
@@ -26,13 +26,7 @@
...
     You also need to confirm that you’re not guilty of misconduct that affects the fairness of the procurement.
 
-    It’s unlikely your application will be accepted if you:
-
-    <ul class="list-bullet">
-    <li>don’t agree (answer ‘no’) to the terms given in questions [[termsOfParticipation]], [[termsAndConditions]], [[canProvideFromDayOne]], [[evidence]], [[10WorkingDays]] and [[MI]]</li>
-    <li>answer ‘yes’ to say you've been involved in the misconduct listed in question [[unfairCompetition]]</li>
-    </ul>
-
+    If you cannot answer ‘yes’ to every question on this page, it is very unlikely that your application will be accepted.
```
[the remark this answers is not in the document]
Comment (NoraGDS) [reply frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml:37]: @bjgill: 'opens in a new tab' please

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml (hunk 2 above)
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml:5]: > `By starting an application for Digital Outcomes and Specialists&nbsp;5 do you understand:`
>
> Is this the correct content? It doesn't quite scan for me :thinking: 

@NoraGDS - any thoughts? Could we shorten this to `Do you understand`?

**Comment by bjgill** [comment]
The functional tests fail quite badly if run against this PR. However, I _think_ that's just a consequence of not having done https://trello.com/c/eLi7y2wX/423-update-functional-tests-if-necessary-ticket-tbc yet

**Comment by lfdebrux** [comment]
I've noticed that we now have `prefill: False` for all sections in the declaration manifest, but we're still asking suppliers if they want to copy their declaration from a previous framework. It would be nice if we could stop that, but that will probably have to be tech debt :/

**Comment by bjgill** [comment]
Agreed - it's not ideal. I'll make a note of the `prefill` problems in the story. If we get time, we can take a look. Otherwise, I agree that this may end up as tech debt.

With luck, less will change in future frameworks, so pre-fill may become meaningful again.

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml [hunk 5]
```diff
@@ -1,8 +1,14 @@
...
+question: |
+  You can read about how to apply in the <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide#how-to-apply" target="_blank" rel="noopener noreferrer">Digital Outcomes and Specialists suppliers’ guide (opens in new tab)</a>.
+
+  By starting an application for Digital Outcomes and Specialists&nbsp;5 do you understand:
+
+  - the application process
+  - that you must make the supplier declaration and mark at least one service as complete before your application can be considered
+  - that you can ask clarification questions only through the updates page in your Digital Marketplace account
+question_advice: >
+  You can ask clarification questions about Digital Outcomes and Specialists&nbsp;5 until 5pm BST on 30 October 2020. All questions and answers will be published regularly on the updates page. You will be notified by email when new questions and answers are available.
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml:11]: Putting this content in the question_advice means that it's not immediately clear what I'm answering yes/no to

Screenshot 2020-09-29 at 15 58 40

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/contactEmail.yml [hunk 6]
```diff
@@ -0,0 +1,3 @@
+name: Contact email
+question: What is the contact’s email address?
+type: text
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/questions/declaration/contactEmail.yml:3]: The DOS5 validator will need to be updated to make sure this field is validated to be an email address.

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml (hunk 5 above)
Comment (NoraGDS) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml:11]: Agree.

```suggestion
question_advice: >
  You can ask clarification questions about Digital Outcomes and Specialists&nbsp;5 until 5pm BST on 30 October 2020. All questions and answers will be published regularly on the updates page. You will be notified by email when new questions and answers are available.
```
Move this to below Yes/No buttons.

Comment (NoraGDS) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml:11]: Move opening paragraph to below heading, so the numbering does not apply to it.

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml (hunk 2 above)
Comment (NoraGDS) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml:5]:
```suggestion
  By starting an application for Digital Outcomes and Specialists 5, do you understand:
```

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/contactEmail.yml (hunk 6 above)
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/contactEmail.yml:3]: :+1: added to the story checklist

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml (hunk 5 above)
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml]: I can move the opening paragraph up. However, I can't easily move the third paragraph down. I can make it into a hint, though. It now looks like:

image

**Review by lfdebrux — APPROVED** [review]
These changes look good to me, thanks @bjgill and @NoraGDS for the big effort!

**Comment by bjgill** [comment]
Thanks Laurence and Nora for the help.

@NoraGDS - any last comments?

Otherwise I'll merge first thing tomorrow.

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml (hunk 5 above)
Comment (NoraGDS) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/readUnderstoodGuidance.yml]: LGTM

**Comment by NoraGDS** [comment]
Looks good. Thank you.
